ptf/platform_helper/brcm_t0_sai_helper.py

```diff
@@ -62,3 +74,196 @@ def start_switch(self):
 
         print("Waiting for switch to get ready, {} seconds ...".format(switch_init_wait))
         time.sleep(switch_init_wait)
+
+
+    def get_port_list(self):
+        """
+        Set the class variable port_list.
+        
+        Output variable:
+            self.port_list
+        """
+        port_list = sai_thrift_object_list_t(count=100)
+        p_list = sai_thrift_get_switch_attribute(
+            self.client, port_list=port_list)
+        self.port_list = p_list['port_list'].idlist
+
+
+    def create_default_route_intf(self):
+        """
+        Create default route interface on loop back interface.
+
+        Output variables:
+            self.default_vrf
+            self.lpbk_intf
+        """
+        print("Create loop back interface...")
+        attr = sai_thrift_get_switch_attribute(self.client, default_virtual_router_id=True)
+        self.default_vrf = attr['default_virtual_router_id']
+        self.assertNotEqual(self.default_vrf, 0)
+        self.lpbk_intf = sai_thrift_create_router_interface(
+            self.client, type=SAI_ROUTER_INTERFACE_TYPE_LOOPBACK,
+            virtual_router_id=self.default_vrf)
+        self.assertEqual(self.status(), SAI_STATUS_SUCCESS)
+
+    
+    def get_default_1q_bridge(self):
+        """
+        Get defaule 1Q bridge.
+
+        Output variables:
+            self.default_1q_bridge_id
+            self.default_vlan_id
+        """
+        print("Get default 1Q bridge...")
+        def_attr = sai_thrift_get_switch_attribute(
+            self.client, default_1q_bridge_id=True, default_vlan_id=True)
+        self.default_1q_bridge_id = def_attr['default_1q_bridge_id']
+        self.default_vlan_id = def_attr['default_vlan_id']
```

Comment:
'default_vlan_id' may can be seperated from this method

Reply:
Yes, that can be.